pybricks/_common.py

```diff
@@ -421,16 +440,37 @@ def off(self):
         """Turns off the light."""
         pass
 
-    def rgb(self, red, green, blue):
-        """Sets the brightness of the red, green, and blue light.
+    def hsv(self, hue, saturation=100, value=100):
+        """Set the hue, saturation and brightness of the light.
 
         Arguments:
-            red (:ref:`brightness`): Brightness of the red light.
-            green (:ref:`brightness`): Brightness of the green light.
-            blue (:ref:`brightness`): Brightness of the blue light.
+            hue (:ref:`angle`): Hue of the color.
```

Comment:
Would be nice to say 0 to 360 here.

Reply:
I think we'll want to add hue to the `signals` page as well, where it can have one of those diagrams.